src/components/club/listing/ClubDetailsCard.tsx

```diff
@@ -0,0 +1,65 @@
+import { BaseCard } from '@src/components/common/BaseCard';
+import { RouterOutputs } from '@src/trpc/shared';
+
+type ClubDetailsCardProps = {
+  club: NonNullable<RouterOutputs['club']['getDirectoryInfo']>;
+  lastEventDate: Date | null;
+};
+export default function ClubDetailsCard({
+  club,
+  lastEventDate,
+}: ClubDetailsCardProps) {
+  return (
+    <BaseCard className="flex flex-col bg-neutral-50 border-slate-200 shadow-sm p-5 gap-2 text-sm text-slate-600">
+      <h2 className="text-xl font-bold text-slate-900 mb-2">Details</h2>
+      {club.numMembers ||
+      club.foundingDate ||
+      lastEventDate ||
+      club.updatedAt ? (
+        <>
+          {club.numMembers && (
+            <div className="flex flex-row w-full justify-between py-1 border-b border-slate-100 last:border-0">
+              <span className="font-medium text-slate-500">Members</span>
+              <span className="text-slate-500">{club.numMembers}</span>
```

Comment:
There should be a unit, the number looks a bit strange by itself IMO
```suggestion
              <span className="text-slate-500">{`${club.numMembers} ${club.numMembers !== 1 ? 'people' : 'person'}`}</span>
```